frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -256,8 +256,8 @@ describe('Kubernetes resource CRUD operations', () => {
       await browser.get(`${appHost}/k8s/cluster/namespaces`);
       // Filter by resource name to make sure the resource is on the first page of results.
       // Otherwise the tests fail since we do virtual scrolling and the element isn't found.
-      await crudView.filterForName(name);
       await crudView.resourceRowsPresent();
+      await crudView.filterForName(name);
```

Comment:
This changes the logic. We want to make sure rows are present *after* filtering. Is there a reason this change was needed?

Reply:
Understood, reverted, and verified FF worked (with `bridge` running on host, instead of inside VM/Docker).